### ISSUE TITLE
Blue Button Color Contrast

### ISSUE THREAD

**Issue by bpmcneilly** [benetech/MathShare#958]
The class `“btn-primary”` presents buttons to users that have insufficient color contrast. The text for these buttons appears in white (#ffffff) on a light blue (#007bff) background, creating a color contrast ratio of 3.98:1, below the 4.5:1 required for normal sized text. The same background color is used when the Sketch button is selected (`class=“scratchActive”`).

#### Suggested Solution
On hover, the buttons darken to #0069d9, which does create a sufficient color contrast ratio (5.22:1 with the white text), adjusting the buttons to always display this shade of blue will ensure a sufficient color contrast ratio is attained at all times.
